### PR TITLE
Upgraded patternlab-node to v3

### PR DESCRIPTION
**Pull request by ganny26**
#42 
I have followed official wiki from patternlab and made changes

https://github.com/pattern-lab/patternlab-node/wiki/Upgrading

Major changes:
* `patternlab-config.json` file has major changes 
*  Package names changed to scoped packages from now

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,6 +9,8 @@
         "@babel/polyfill": "^7.4.0",
         "@babel/preset-env": "^7.4.2",
         "@babel/register": "^7.4.0",
+        "@pattern-lab/patternengine-node-mustache": "^2.0.0-alpha.2",
+        "@pattern-lab/patternlab-node": "3.0.0-alpha.8",
```
Comment (mbulfair) [inline package.json]: make sure to remove "patternlab-node": "^2.12.0",

**Comment by ganny26** [comment]
Hi @mbulfair 

* Removed old `patternlab-node` 
* Also upgraded `styleguidekit-assets-default` and `styleguidekit-mustache-default` to new scoped packages 
* Post install script not required as mentioned in upgrading [wiki](https://github.com/pattern-lab/patternlab-node/wiki/Upgrading)

Now project building successfully

File: package.json (hunk 1 above)
Comment (ganny26) [reply package.json]: Removed in recent commit

**Comment by mbulfair** [comment]
@ganny26 

This warning appears:
```
Configuration key [debug] inside patternlab-config.json was found. As of Pattern Lab Node Core 3.0.0 this key is replaced with a new key, [logLevel]. This is a string with possible values ['debug', 'info', 'warning', 'error', 'quiet'].
Turning on 'info', 'warning', and 'error' levels by default, unless [logLevel] is present. If that is the case, [debug] has no effect.
```
as well as ,

```
(node:10561) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
```

**Pull request closed without merging**